tests/Unit/Emogrifier/CssConcatenatorTest.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+namespace Pelago\Tests\Unit\Emogrifier;
+
+use Pelago\Emogrifier\CssConcatenator;
+
+/**
+ * Test case.
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ */
+class CssConcatenatorTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var CssConcatenator
+     */
+    private $subject = null;
+
+    /**
+     * @return void
+     */
+    protected function setUp()
+    {
+        $this->subject = new CssConcatenator();
+    }
+
+    /**
+     * @test
+     */
+    public function getCssInitiallyReturnsEmptyString()
+    {
+        $result = $this->subject->getCss();
+
+        static::assertSame('', $result);
+    }
+
+    /**
+     * @test
+     */
+    public function getCssReturnsSingleAppendedRule()
```

Comment:
I think the test names should focus on what `append` does, not what `getCss` does (as `append` is what does the work). In this case, I propose the name `appendSetsFirstRule` (or something similar).